lua/astrocommunity/code-runner/molten-nvim/init.lua

```diff
@@ -0,0 +1,37 @@
+---@type LazySpec
+return {
+  "benlubas/molten-nvim",
+  cmd = {
+    "MoltenInfo",
+    "MoltenInit",
+    "MoltenDein",
+    "MoltenGoto",
+    "MoltenNext",
+    "MoltenPrev",
+    "MoltenEvaluateLine",
+    "MoltenEvaluateVisual",
+    "MoltenEvaluateOperator",
+    "MoltenEvaluateArgument",
+    "MoltenReevaluateCell",
+    "MoltenDelete",
+    "MoltenShowOutput",
+    "MoltenHideOutput",
+    "MoltenEnterOutput",
+    "MoltenInterrupt",
+    "MoltenOpenInBrowser",
+    "MoltenImagePopup",
+    "MoltenRestart",
+    "MoltenSave",
+    "MoltenLoad",
+    "MoltenExportOutput",
+    "MoltenImportOutput",
+  },
+  version = "^1.0.0", -- use version <2.0.0 to avoid breaking changes
```

Comment:
```suggestion
  version = "^1", -- use version <2.0.0 to avoid breaking changes
```

unbelievably minor nit, take it or leave it haha